packages/cozy-search/src/components/Search/AssistantButton.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import IconButton from 'cozy-ui/transpiled/react/IconButton'
+import InputAdornment from 'cozy-ui/transpiled/react/InputAdornment'
+import { useCozyTheme } from 'cozy-ui/transpiled/react/providers/CozyTheme'
+
+import { TwakeAssistantIcon } from '../AssistantIcon/TwakeAssistantIcon'
+
+export const AssistantButton = ({ onClick, size }) => {
+  const { type } = useCozyTheme()
+
+  return (
+    <InputAdornment position="end" className="u-mr-half">
+      <IconButton onClick={onClick} size="small">
+        <Icon
+          color={type === 'light' ? 'var(--primaryColor)' : 'var(--white)'}
```

Comment:
we should not do this, please report to Swann this problematic to avoid it as much as possible

Reply:
Fixed